src/studio/src/designer/frontend/ux-editor/App.tsx

```diff
@@ -27,49 +30,18 @@ export function App() {
   const fetchFiles = () => {
     const { org, app } = window as Window as IAltinnWindow;
     const appId = `${org}/${app}`;
+    dispatch(fetchDataModel());
+    dispatch(FormLayoutActions.fetchFormLayout());
 
-    // ALTINN STUDIO
-    // Editor
-    // Set design mode to true
-    appDataActionDispatcher.setDesignMode(true);
-
-    // Fetch data model
-    appDataActionDispatcher.fetchDataModel(
-      `${window.location.origin}/designer/${appId}/Model/GetJson`,
-    );
-    // Fetch form layout
-    formDesignerActionDispatchers.fetchFormLayout(
-      `${window.location.origin}/designer/${appId}/UIEditor/GetFormLayout`,
-    );
-    // Load text resources
     const languageCode = 'nb';
-    const url = `${window.location.origin}/designer/${appId}/UIEditor/GetTextResources/${languageCode}`;
-    dispatch(loadTextResources({ url }));
+    dispatch(loadTextResources({ url: getLoadTextResourcesUrl(languageCode) }));
+    dispatch(fetchServiceConfiguration());
+    dispatch(fetchRuleModel());
 
-    // Fetch ServiceConfigurations
-    manageServiceConfigurationActionDispatcher.fetchJsonFile(
-      `${window.location.origin}/designer/${
-        appId}/UIEditor/GetJsonFile?fileName=RuleConfiguration.json`,
-    );
-    // Fetch rule connections
-    appDataActionDispatcher.fetchRuleModel(
-      `${window.location.origin}/designer/${appId}/UIEditor/GetRuleHandler`,
-    );
-    // Fetch the CodeLists
-    appDataActionDispatcher.fetchCodeLists(
-      `${window.location.origin}/designer/${appId}/CodeList/CodeLists`,
-    );
-    // Fetch thirdParty Components
-    ThirdPartyComponentsActionDispatcher.fetchThirdPartyComponents(
-      `${window.location.origin}/designer/${appId}/UIEditor/GetThirdPartyComponents`,
-    );
-    // Fetch language
-    appDataActionDispatcher.fetchLanguage(
-      `${window.location.origin}/designerapi/Language/GetLanguageAsJSON`, 'nb',
-    );
-    // Fetch widget settings
+    const location = `${window.location.origin}/designer/${appId}/UIEditor/GetThirdPartyComponents`;
+    dispatch(fetchThirdPartyComponents({ location }));
```

Comment:
Is third party components ever used? Can probably be removed entirely.

Reply:
Nope. I agree, will remove.